README.md

```diff
@@ -33,6 +33,8 @@ Inside a fresh virtualenv, `cd` into the root folder of this repository (`xblock
 ```bash
 $ pip install -U pip wheel
 $ pip install -r requirements-test.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/base.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/test.txt
```

Comment:
I guess the new problem-builder requirement is somehow responsible for this. Don't worry about it, it's fine to leave these two additional `pip install` steps in.